tests/logging_configurator_tests.py

```diff
@@ -0,0 +1,55 @@
+# licensed to the apache software foundation (asf) under one
```

Comment:
this is using all lowercase, will probably fail the license check

Reply:
weird, I copied from other files... I wonder if something in pre-commit did that? Oh well, good catch. Thanks. Fixed.